_episodes/17-cluster.md

```diff
@@ -170,40 +170,47 @@ $ pip install --user snakemake
 Assuming you've transferred your files and everything is set to go,
 the command `snakemake -n` should work without errors.
 
-## Cluster configuration with `cluster.json`
-
-Snakemake uses a JSON-formatted configuration file to retrieve cluster submission parameters.
-An example (using SLURM) is shown below.
-You can check to see if your `cluster.json` is valid JSON syntax by pasting its contents into
-the box at [jsonlint.com](https://jsonlint.com).
-
-```json
-{
-    "__default__":
-    {
-        "account": "a_slurm_submission_account",
-        "mem": "1G",
-        "time": "0:5:0"
-    },
-    "count_words":
-    {
-        "time": "0:10:0",
-        "mem": "2G"
-    }
-}
+## Creating a cluster profile
+
+Snakemake uses a YAML-formatted configuration file to retrieve cluster submission parameters; we will use the SLURM scheduler for an example. When we use the '--profile slurm' argument, snakemake looks for a directory with the name of our profile (slurm) containing a 'config.yaml' file such as the one below.
+
+```yaml
+cluster: "sbatch --time={resources.time_min} --mem={resources.mem_mb} -c {resources.cpus} -o slurm/logs/{rule}_{wildcards} -e slurm/logs/{rule}_{wildcards}"
```

Comment:
Long one-liners are best avoided in introductory material.
The [example for Slurm](https://github.com/Snakemake-Profiles/slurm/blob/master/%7B%7Bcookiecutter.profile_name%7D%7D/config.yaml) specifies a normal `sbatch` script, which would define the appropriate settings and greatly de-clutter this YAML file.